tests/5.1/runtime_calls/test_omp_display_env.c

```diff
@@ -0,0 +1,27 @@
+//---------test_omp_display_env.c-------------------
+//
+//OpenMP API Version 5.1 Aug 2021
+//
+//Tests the behavior of omp_display_env. This test 
+//should print out the OpenMP version number and 
+//the initial values of the ICV's associated with 
+//the environment variables.
+//------------------------------------------------
```

Comment:
lgtm, i would adjust the file header a bit.
i would add some equal signs before and after the dashes on lines 1 & 9, add a space after each `//`, and and an empty line between lines 8 & 9. (https://crpl.cis.udel.edu/ompvvsollve/documentation/codingstandards/)